client/src/modules/reports/generate/inventory_changes/inventory_changes.config.js

```diff
@@ -56,5 +60,10 @@ function inventoryChangesController($sce, Notify, SavedReports, AppCache, report
     if (cache.reportDetails) {
       vm.reportDetails = angular.copy(cache.reportDetails);
     }
+
+    // Set the defaults
+    if (!angular.isDefined(vm.reportDetails.currencyId)) {
```

Comment:
It's probably best to just put this on line 15.

Reply:
Ok